src/components/mobile-docs-sidebar/index.tsx

```diff
@@ -0,0 +1,53 @@
+import Sidebar, { SidebarProps } from 'components/sidebar'
+import { useState } from 'react'
+
+interface MobileDocsSidebar {
+  levels: SidebarProps[]
+}
+
+const MobileDocsSidebar = ({ levels }: MobileDocsSidebar) => {
+  const numberOfLevels = levels.length
+  const [currentLevel, setCurrentLevel] = useState<number>(numberOfLevels - 1)
+  const currentLevelSidebarProps = levels[currentLevel]
+
+  let levelButtonProps
+  if (currentLevel === 0 && numberOfLevels > 1) {
```

Comment:
Similar to what Bryce mentioned above, we should make these level button rendering behavior adaptive, as Learn may have a different number of levels to navigate through.

Reply:
Absolutely! This is super hard-coded right now. Plan is to start abstracting since it sounds like the approach so far is pretty solid. 😊

Reply:
Removed the hard-coded stuff and this file, resolving! ✨